en/salesforce/src/docs/ms_composer_release_notes_2023.adoc

```diff
@@ -11,6 +11,10 @@ See xref:ms_composer_release_notes_ki.adoc[Known Issues for MuleSoft Composer fo
 
 include::_partials/release-notes/rn-headings.adoc[tag=whatsNew,leveloffset=+3]
 
+* You can now connect to two new system: Sage Intaact. +
+
+See xref:ms_composer_reference.adoc[the product documentation] for details about each connector.
```

Comment:
This is picky but it is better to have the info on why you would click a link before the link itself:

For details about each connector, see...

Reply:
I will consider this - this has been done the same way since GA, so they would all need to be changed.